test/integration/integration_test.js

```diff
@@ -0,0 +1,207 @@
+describe('The data layer helper library', function() {
+  it('does not pollute the global scope', function() {
+    expect(window['dataLayer']).toBe(undefined);
```

Comment:
Could you please take a look to migration of these tests to Jasmine's toBeUndefined()/spy() etc?

Reply:
I will put it in a different commit